core/src/main/java/org/apache/accumulo/core/metadata/schema/Ample.java

```diff
@@ -296,7 +296,7 @@ public interface ConditionalTabletsMutator extends AutoCloseable {
      * @return A fluent interface to conditional mutating a tablet. Ensure you call
      *         {@link ConditionalTabletMutator#submit(RejectionHandler)} when finished.
      */
-    OperationRequirements mutateTablet(KeyExtent extent);
+    OperationRequirements mutateTablet(KeyExtent extent, Text prevEndRow);
```

Comment:
There would be a lot less redundancy in the code if the prevEndRow was not passed and the prev endRow form the extent was used.  Also I am not sure they should ever be different.